sdk/sui-owned-object-pools/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "suioop",
+  "version": "2.3.2",
+  "description": "Sui Owned Object Pools is a library that provides a set of tools for managing multiple concurrent transactions on the Sui network to help avoiding object equivocation and locking",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "source": "./src/index.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
+  "files": [
+    "dist",
+    "src",
+    "CHANGELOG.md"
+  ],
+  "scripts": {
+    "format": "prettier --write '**/*.{ts,tsx,js,jsx,json,css,md}'",
+    "lint": "eslint --ext .ts --fix --max-warnings 0 src",
+    "test": "jest test/unit/executorServiceHandler.test.ts && jest test/unit/pool-tx-execution.test.ts && jest test/unit/pool-init.test.ts",
+    "release": "pnpm build:types && pnpm build:tsup && release-it",
```

Comment:
releases should be handled through our normal changesets flow (this should work automatically without a specific script for it)